lib/takeaway.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'order'
+require_relative 'dish_list'
+require_relative 'menu'
+require_relative 'display_output'
+require_relative 'twilio_api'
+
+class Takeaway
+  include DisplayOutput
+  CURRENT_LEAD_TIME_IN_MINS = 60
+  def initialize
+    @order = Order.new
```

Comment:
a little odd that takeaway only ever has one order, but not the end of the world - I guess as long as it gets refreshed when the next customer comes along :-)